Commands.adoc

```diff
@@ -0,0 +1,239 @@
+
+Similarly for OpenShift you can use: +
+`$ vagrant service-manager env openshift`
+
+If you want to run the plugin and get the environment variables and certificates
+ for docker in key=value format, use:
+----------------------------------------------------------------------
+$ vagrant service-manager env docker --script-readable
+DOCKER_HOST=tcp://172.28.128.3:2376
+DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker'
+DOCKER_TLS_VERIFY=1
+DOCKER_API_VERSION=1.21
+----------------------------------------------------------------------
+
+
+== Identify IP Address and Version of the VM
+
+If you need to find information related to the VM, such as the routable
+IP address or the version and release information, use:
+
+`$ vagrant service-manager box` +
+
+Usage: `vagrant service-manager box [sub-command] [options]`
+
+The following sub-commands can be used: +
+`ip`: Displays the routable IP address of the VM +
+`version`: Displays the version and release information of the VM +
+
+The following options can be used: +
+`--script-readable`: Displays information in a key=value format +
+`-h, --help`: Prints help for the specific command being run.
+
+=== Examples
+
+To check the routable IP address of the VM:
+---------------------------------
+$ vagrant service-manager box ip
+172.28.128.3
+---------------------------------
+
+To check the routable IP address of the VM in key=value format:
+---------------------------------------------------
+$ vagrant service-manager box ip --script-readable
+IP=172.28.128.3
+---------------------------------------------------
+
+To check the version of the VM:
+--------------------------------------
+$ vagrant service-manager box version
+Atomic Developer Bundle (ADB) 2.2.0
+--------------------------------------
+
+To check the version of the VM in key=value format:
+--------------------------------------------------------
+$ vagrant service-manager box version --script-readable
+VARIANT="Atomic Developer Bundle (ADB)"
+VARIANT_ID="adb"
+VARIANT_VERSION="2.2.0"
+--------------------------------------------------------
+
+
+== Manage Service Lifecycles
+
+If you need to mange the lifecycle of a service - check the current status of
+existing services, start or stop or restart a specific service use: +
+`$ vagrant service-manager [sub-command] [service]`
+
+The following sub-commands can be used: +
+`status`:   Lists services and their state (running/stopped) +
+`start`:   Starts the given service in the VM +
+`stop`:   Stops the given service in the VM +
+`restart`:   Restarts the given service in the VM +
+
+The possible options for service are `docker`, `openshift` and `kubernetes`.
+
+=== Examples
+
+If no service is specified `vagrant service-manager status` will provide
+information on all the configured services and their state (running/stopped).
+---------------------------------------
+$ vagrant service-manager status
+Configured services:
+docker - stopped
```

Comment:
hmm, thought the output of the command would be aligned, but apparently is is not :-(. Something we should fix.